### PR TITLE
Add Brazilian Portuguese translation

### PR DESCRIPTION
**Pull request by mathessaricci**

### CODE REVIEW

File: po/pt_BR.po [hunk 1]
```diff
@@ -0,0 +1,360 @@
...
+msgid_plural "%d characters with spaces"
+msgstr[0] "%d caractere com espaços"
+msgstr[1] "%d caracteres com espaços"
+
+#: src/Widgets/ActionBar.vala:61
+#, c-format
+msgid "%d character without spaces"
+msgid_plural "%d characters without spaces"
+msgstr[0] "caractere sem espaços"
+msgstr[1] "caracteres sem espaços"
```
Comment (ryonakano) [inline po/pt_BR.po:258]:
```suggestion
msgstr[0] "%d caractere sem espaços"
msgstr[1] "%d caracteres sem espaços"
```

Seems like you're missing `%d` called Format Specifier here. Maybe should be like this? (I don't understand Portuguese 😅)

**Review by ryonakano — CHANGES_REQUESTED** [review]
Just one changes request and it's ready to go.

**Comment by ryonakano** [comment]
BTW it is recommended to use a translation tool app like [Poedit](https://flathub.org/apps/details/net.poedit.Poedit) or [Lokalize](https://flathub.org/apps/details/org.kde.lokalize) to edit this kind of files, editing header info manually may be a little bit difficult for us :)

**Comment by mathessaricci** [comment]
I'll make the necessary changes. It's my first time translating I'm so sorry for my mistakes. Thank you for being so polite and patient.

**Comment by ryonakano** [comment]
Don't be sorry, doing this kind of thing—translating apps and creating Pull Requests for a couple of times you would learn and get accustomed😉

**Pull request closed without merging**